src/actions/actions.js

```diff
@@ -17,5 +18,8 @@ export const invalidateLocation = (location) => ({
 });
 
 export const fetchForecast = options => dispatch => {
-  // return fetch API call
 };
+
+export const testAction = () => {
```

Comment:
(Part Three)

This is getting you yelled at because the curly braces have to means: object literal syntax and then they also denote a function body. You also don't need the quotes because you made that constant, right?

```js
export const testAction = () => {
  return { type: TEST_ACTION }
}
```

```js
export const testAction = () => ({ type: TEST_ACTION });
```

Either will work. I think I prefer the first.